src/app/home/home.component.html

```diff
@@ -1,12 +1,21 @@
-<ion-header>
-  <ion-toolbar>
-    <ion-title>Home</ion-title>
-  </ion-toolbar>
-</ion-header>
 
 <ion-content padding>
-	<ion-button color="success">This is for test</ion-button>
-  <ion-button color="warning">This is for test</ion-button>
-  This is the start of Practera App V2
-  <p>If you get lost, the <a target="_blank" rel="noopener" href="https://ionicframework.com/docs">docs</a> will be your guide.</p>
+  <h1 text-center>Program Name</h1>
+  <div text-center padding >
+    <circle-progress 
+    [percent]="85"
+    [radius]="100"
+    [outerStrokeWidth]="16"
```

Comment:
Since we have defined these attributes in `home.module` don't need to pass them here again.
Adjust them properly in `home.module` instead